index.html

```diff
@@ -484,6 +490,115 @@
               layout="[[layoutClass]]"
               route="{{subRoute}}"></generic-page>
           </template>
+          <!-- LURES    ============================================   -->
```

Comment:
I absolutely want your help and advice.  Thank you.  I'd like to show you my idea on how to incorporate the legacy boards, lures and upcoming MB3 and definitely get your POV on it.

Maybe you already have a plan for things, which would be fantastic.  I just want things to work for the MB project and am not married to anything except simplicity.